### ISSUE TITLE
Feature: media queries

### ISSUE THREAD

**Issue by manvankaliber** [UltraCakeBakery/cssified-html#10]
I've decided to not include my first implementation of media queries into the codebase since it was very ugly and buggy. In the current codebase there is nothing related to media queries going on, so we have free range of this implementation.

# Syntax
I think the most straight forward syntax would be something like this:

`<div background="#FFF" @dark:background="#000">`. 

and if you want to stack them you can do that through:

`<div background="#FFF" @dark:background="#070707" @dark:@high-contrast:background="#000">`.

# Out-of-the-box supported media queries
Just to name a few:

- `@light` - prefers-color-scheme light
- `@dark` - prefers-color-scheme dark
- `@width-<integer>` - where container width is larger than <integer>
- `@width--<integer>` - where container width is smaller than <integer> 
- `@height-<integer>` - where container height is larger than <integer>
- `@height--<integer>` - where container height is smaller than <integer>
- `@screen-width-<integer>` - where screen width is larger than <integer>
- `@screen-width--<integer>` - where screen width is smaller than <integer> 
- `@screen-height-<integer>` - where screen height is larger than <integer>
- `@screen-height--<integer>` - where screen height is smaller than <integer>
- `@print` - when printing
- `@landscape` - when in landscape mode
- `@portrait` - when in portrait mode
The media queries are configured in the config, and we ship some default ones out of the box that are commonly used.

# TL:DR;
Out of the box we configure some default media queries. Our selector-builder should be responsible for stacking the values correctly. Maybe later we can introduce `and` and `or` syntax. For now this should be perfectly fine though.

**Comment by UltraCakeBakery** [comment]
Sorry for the accidental mention everyone! Forgot to wrap things in quotes 😅
